packages/fennel/build.sh

```diff
@@ -2,9 +2,11 @@ TERMUX_PKG_HOMEPAGE="https://fennel-lang.org"
 TERMUX_PKG_DESCRIPTION="A Lisp that compiles to Lua"
 TERMUX_PKG_LICENSE="MIT"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=1.4.2
+TERMUX_PKG_VERSION=1.5.1
 TERMUX_PKG_SRCURL="https://github.com/bakpakin/Fennel/archive/refs/tags/$TERMUX_PKG_VERSION.tar.gz"
-TERMUX_PKG_SHA256=b44a205ee7ebdee22f83d2a7a87742172295b8086b5361850dfab4f49699e44f
+TERMUX_PKG_SHA256=7456737a2e0fc17717ea2d80083cfcf04524abaa69b1eb79bded86b257398cd0
 TERMUX_PKG_DEPENDS="lua53"
 TERMUX_PKG_PLATFORM_INDEPENDENT=true
 TERMUX_PKG_BUILD_IN_SRC=true
+export LUA_VERSION=5.3
+export LUA=lua5.3
```

Comment:
Why are we compiling `fennel` against Lua 5.3 anyway?
That doesn't seem to be what upstream's Makefile defaults to.

Reply:
I had actually noted this for a future followup to ask about independent of version bumping! I wasn't sure if Lua 5.3 was the default version in termux or something, but if that was the case you could probably just depend on `lua`. I'm glad you called this out, because I forgot to mention it in my PR 😄

Reply:
According to the Git Blame it was introduced like that and never changed.
https://github.com/termux/termux-packages/pull/11942/files

Reply:
With your `TERMUX_PKG_DEPENDS` above, I can just drop `LUA_VERSION` and let the makefile resolve it automatically. I'll just need to tweak it so `LUA` is set in a way that resolves the correct executable provided by `TERMUX_PKG_DEPENDS`. I'll need to go through the docs for finding the best way to approach that, but any suggestions are welcome 😃